portal/server/controllers/dashboard/deals/deals-filters-query.js

```diff
@@ -36,7 +36,17 @@ const dashboardDealsFiltersQuery = async (filters, user, userToken) => {
       const checkersApprovalAmendmentDealIds = await getCheckersApprovalAmendmentDealIds(userToken);
 
       if (checkersApprovalAmendmentDealIds?.length) {
-        orQuery.OR.push({ [CHECKERS_AMENDMENTS_DEAL_ID]: checkersApprovalAmendmentDealIds });
+        /**
+         * add and query to the orQuery
```

Comment:
```suggestion
         * Add an `and` query to the `orQuery`
```

Reply:
Trying to understand the commen, are you saying only deal with status `Acknowledged` will appear on checkers portal application preview page as long as the amendment status is also `READY_FOR_CHECKERS_APPROVAL`?